internal/controller/clusterconnect_controller.go

```diff
@@ -81,6 +82,10 @@ var (
 	}
 )
 
+var (
+	clusterConnectConnectionProbeTimeout = 5 * time.Minute
```

Comment:
Read from the Helm value

Reply:
done here https://github.com/open-edge-platform/cluster-connect-gateway/pull/41/commits/93d475d3ebc482ee0919a12d9ab9642d6cd71ffc